packs/feats/golem-dynamo.json

```diff
@@ -48,6 +48,15 @@
                 "selector": "saving-throw",
                 "type": "status",
                 "value": 1
+            },
+            {
+                "definition": [
+                    "item:slug:sterling-dynamo"
```

Comment:
Can you confirm this works in game?

Reply:
Yep, tested and it works.
<img width="291" alt="Screen Shot 2023-11-06 at 3 21 48 PM" src="https://github.com/foundryvtt/pf2e/assets/1904698/1bc6feb5-2276-4411-9d93-e4e9a41e3f11">